### PR TITLE
Create README.md with Installation instructions

### PR DESCRIPTION
**Pull request by NHDaly**
For brand new would-be julia users who want to get started with Julia, I would love to send them to this github repo to get started, but there's no information on this page explaining _what to do once you're here!_

I know that these tutorials were originally written with JuliaBox in mind, so i've tried to maintain that spirit in these instructions. But hopefully we can also include instructions here about how to get started with julia in _any capacity_, because JuliaBox may not be the preferred option for everyone, and these are great tutorials that it would be a shame to not share! :)

CC: @xorJane

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+   - Follow the instructions to install it on your computer.
+2. Install julia's Jupyter Notebooks integration: IJulia.jl
+   - Open julia, and you are presented with a "REPL" prompt. This is the main Julia interface. There, type this closing bracket
+     character: <kbd>]</kbd> to open the package manager. Then type `add IJulia` to install the jupyter notebook interface for julia.
+   - Then exit the package manager by pressing <kbd>delete</kbd> (as if you're deleting the `]` you typed to enter package mode)
+   - Now you can open the jupyter notebooks by entering `using IJulia`, then once that loads, entering `IJulia.notebook()`, which should
+     open a Jupyter tab in your browser.
+3. Last, download the tutorials from this repository, via the github Clone/Download button above, or clicking this link:
+    - https://github.com/JuliaComputing/JuliaBoxTutorials/archive/master.zip
+    - (If you've never used GitHub before, it's a place to collaborate on open source software. Julia itself is also stored on github!: https://github.com/JuliaLang/julia)
```
Comment (mbauman) [inline README.md]:
```suggestion
    - (If you've never used GitHub before, it's a place to collaborate on open source software. Julia itself is also [developed on github!](https://github.com/JuliaLang/julia))
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Installing on your Computer
+
+You can also do these tutorials by installing `julia` on your computer, setting up Jupyter, and downloading this tutorial repository to your computer.
+If you're new to Julia, you can do that by following these steps:
+
+1. Download julia from https://julialang.org/downloads/ (download the latest "stable" version).
+   - Follow the instructions to install it on your computer.
+2. Install julia's Jupyter Notebooks integration: IJulia.jl
+   - Open julia, and you are presented with a "REPL" prompt. This is the main Julia interface. There, type this closing bracket
```
Comment (mbauman) [inline README.md]: How do I open julia?  Are we okay punting on this point?

Comment (NHDaly) [reply README.md]: Mmm good point.
Do you think the changes in d376d0eef65594ee62bda1f9af68d5c624fb48c0 make this any clearer?

Comment (xorJane) [reply README.md]: I think so!

**Pull request merged**